pipeline/data_merging/brca_pseudonym_generator.py

```diff
@@ -100,6 +103,9 @@ def compute_genomic_hgvs(cDNA, assemblyMapper):
     except HGVSError as e:
         logging.info("Exception during conversion of " + str(cDNA) + " to genomic coordinates: " + str(e))
         return None
+    except AttributeError as e:
```

Comment:
I think it would be a cleaner fix to just test if cdna is None at the beginning of the function. This makes it clear that we expect that `cDNA` can be `None` and how we handle this case. (you get this AttributeError whenever `cDNA` is `None`)